explorer/src/components/layout/DomainHeader.tsx

```diff
@@ -19,23 +19,24 @@ export const DomainHeader = () => {
   const [isOpen, setIsOpen] = useState(false)
   const { network } = useChains()
 
-  const menuList = useMemo(
-    () => [
+  const menuList = useMemo(() => {
+    const domainsChildren = ROUTES.find((item) => item.name === Routes.domains)?.children
+    const list = [
       {
         title: 'Domains',
         link: `/${network}/${Routes.domains}`,
       },
-      {
-        title: 'Nova',
-        link: `/${network}/${Routes.nova}`,
-      },
-      {
-        title: 'Auto-ID',
-        link: `/${network}/${Routes.autoid}`,
-      },
-    ],
-    [network],
-  )
+    ]
+    if (domainsChildren)
```

Comment:
wouldn't be cleaner to use an Array.map function here?

Reply:
Something like:

```js
[
      {
        title: 'Domains',
        link: `/${network}/${Routes.domains}`,
      },
      ...domainsChildren.stuff
 ]
```

Reply:
Yes thanks